service_agent/package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "debug": "DEBUG=1 nodemon",
     "start": "nodemon",
+    "start:debug": "nodemon --watch \"src/*.ts\" \"src/**/*.ts\" \"node_modules/*jolocom*/**/*.js\" --ext ts,js --ignore src/**/*.spec.ts --exec \"node --inspect -r ts-node/register ./src/index.ts --inspect\"",
```

Comment:
There's already a nodemon.json file that contains most of this configuration, please use that